src/models/genezio-models.ts

```diff
@@ -0,0 +1,195 @@
+import { YamlClassConfiguration, YamlMethodConfiguration } from "./yamlProjectConfiguration";
+
```

Comment:
the name of the file is also very generic and I think it contains multiple types and classes from different domains: File, AST models and Ast Generator models. It should definitely be splited into multiple files.